### PR TITLE
multus, static policy: do not follow tagged multus versions

### PR DESCRIPTION
**Pull request by maiqueb**
Signed-off-by: Miguel Duarte Barroso <mdbarroso@redhat.com>

**What this PR does / why we need it**:

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

File: hack/components/bump-multus.sh [hunk 1]
```diff
@@ -123,11 +123,9 @@ cp ${MULTUS_PATH}/config/cnao/000-ns.yaml data/multus/
 cp ${MULTUS_PATH}/config/cnao/001-multus.yaml data/multus/
 
 echo 'Get multus image name'
-MULTUS_TAG=$(git-utils::get_component_tag ${MULTUS_PATH})
+MULTUS_TAG="latest-thick"
 MULTUS_IMAGE=ghcr.io/k8snetworkplumbingwg/multus-cni
-# TODO: do not use this hardcoded image, and rely on tagged versions instead.
-# had to resort to this shenanigan because of https://github.com/k8snetworkplumbingwg/multus-cni/issues/944
-MULTUS_IMAGE_TAGGED=${MULTUS_IMAGE}@sha256:09a572e8bdf8a398db024ca252d06cf3ac0a03e07ae547d6a84221d4f6a9f96f
```
Comment (maiqueb) [inline hack/components/bump-multus.sh:130]: @RamLavi do you think it would be honest enough to put this back the way it was **but** update the image SHA ?

Comment (RamLavi) [reply hack/components/bump-multus.sh:130]: I don't follow

Comment (maiqueb) [reply hack/components/bump-multus.sh:130]: forget it. I'm simply updating the image to a thick plugin version that works for anything requiring resources.

**Comment by RamLavi** [comment]
/lgtm
/approve

@maiqueb can you rebase?

**Comment by maiqueb** [comment]
@RamLavi please grant the lgtm tag again :)

**Review by RamLavi — APPROVED** [review]
/lgtm

**Pull request merged**